core-backend/sandbox/src/funcs.rs

```diff
@@ -120,11 +122,13 @@ impl<E: Ext + 'static> FuncsHandler<E> {
             let dest: ProgramId = funcs::get_bytes32(memory, program_id_ptr)?.into();
             let payload = funcs::get_vec(memory, payload_ptr, payload_len)?;
             let value = funcs::get_u128(memory, value_ptr)?;
-            ext.send(HandlePacket::new(dest, payload, value))
-                .map_err(FuncError::Core)
-                .on_success_code(|message_id| wto(memory, message_id_ptr, message_id.as_ref()))
+            Ok(ExtErrorProcessor::new(ext)
+                .with(|ext| ext.send(HandlePacket::new(dest, payload, value)))
+                .map_err(FuncError::Core)?
+                .on_success(|message_id| wto(memory, message_id_ptr, message_id.as_ref()))?
+                .error_len())
         })
-        .map(|code| Value::I32(code).into())
+        .map(|code| Value::I32(code as i32).into())
         .map_err(|err| {
```

Comment:
Could we directly store error to ctx field from extErrorSaving implementation?

Reply:
What error?

Reply:
For this case, for example, `.map_err(FuncError::Core)?`

Reply:
this ext implementation already saved this error in memory, isn't it?

Reply:
Yes, it did

Reply:
`ErrorSavingExt` saves only `ExtError`. The rest errors are thrown and saved in `ctx`